tests/integration/Paginator/Adapter/QueryBuilderCest.php

```diff
@@ -199,6 +199,6 @@ public function testIssue13552(IntegrationTester $I)
         ))->paginate();
 
         $I->assertEquals(1, $paginate->last);
-        $I->assertEquals(4, $paginate->total_items);
+        $I->assertEquals(1, $paginate->total_items);
```

Comment:
Did you change the business logic?

Reply:
QueryBuilder returns only one row, since there are only three entries in the test table.

Reply:
Fixed. I was wrong about the records in the table